e2e/utils.go

```diff
@@ -1266,3 +1274,38 @@ func validateSubvolumegroup(f *framework.Framework, subvolgrp string) error {
 	}
 	return nil
 }
+
+// upgradeCSI deploys a desired ceph-csi release version.
+func upgradeCSI(version string) error {
```

Comment:
Move this upgrade functions to upgrade.go

Reply:
That file will have only 2 functions.. Will that be fine?

Reply:
Yeah sure

Reply:
Okay, I will create a separate file for them.